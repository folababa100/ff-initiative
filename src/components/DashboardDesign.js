import React from 'react';
import DashboardMain from "./DashboardMain";
// import DashboardFooter from "./DashboardFooter";

const DashboardDesign = () => {
  return (
    <div className="back-color">
      <DashboardMain/>
      {/* <DashboardFooter/> */}
    </div>
  )
}

export default DashboardDesign;
