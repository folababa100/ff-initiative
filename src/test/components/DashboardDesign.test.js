import React from 'react';
import { shallow } from "enzyme";
import DashboardDesign from '../../components/DashboardDesign';

test('should correctly render component correctly', () => {
  const wrapper = shallow(<DashboardDesign/>);
  expect(wrapper).toMatchSnapshot();
})

