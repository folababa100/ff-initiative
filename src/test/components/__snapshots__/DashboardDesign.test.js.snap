// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DashboardDesign />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Connect(DashboardMain) />,
      "className": "back-color",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {},
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Connect(DashboardMain) />,
        "className": "back-color",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
